pkgs/development/libraries/g2o/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, cmake, eigen, suitesparse }:
+
+stdenv.mkDerivation {
+  pname = "g2o-unstable";
```

Comment:
After doing a quick search it appears most packages put the `-unstable` bit in the version, like so:
```
pname = "g2o";
version = "unstable-2019-04-07";
```

That being said... the nixpkgs manual explicitly says that the version string should start with a digit. I'm guessing the manual needs to be updated to reflect changes since `pname` became standard.

How about you go with what I've mentioned above.